dayz-launcher.sh

```diff
@@ -202,7 +202,7 @@ dec2base64() {
       }
     ' \
     | base64 \
-    | sed 's|/|-|g; s|=||g'
+    | sed 's|/|-|g; s|=||g; s|+|_|g'
```

Comment:
Could you please move your addition so that the `=` suffix removal stays last? That keeps the intention clear.

```suggestion
    | sed 's|/|-|g; s|+|_|g; s|=||g'
```